src/styles/components/cards.scss

```diff
@@ -0,0 +1,119 @@
+.stars-line {
+  display: flex;
+}
+
+.stars-line__star {
```

Comment:
It's separate BEM block, it should be in separate scss file
Also, use scss nesting